### PR TITLE
feat(typography): add Blockquote component example, configuration, and API

### PR DESCRIPTION
**Pull request by jzempel**
## Description

See the `/components/typography#blockquote` URL for review.

## Checklist

- [x] :ok_hand: website updates are Garden Designer approved (add the designer as a reviewer)
- [x] :black_nib: copy updates are approved (add the content strategist as a reviewer)
- [x] :link: considered opportunities for adding cross-reference URLs (grep for keywords)
- [x] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [x] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

**Review by steue — APPROVED** [review]
looks good to me! the garden related quotes fit well. i wonder if v can sprinkle some magic spice?

File: src/pages/components/typography.mdx [hunk 1]
```diff
@@ -16,6 +17,17 @@ components:
 
 ## How to use it
 
+### Block quotation
```
Comment (allisonacs) [inline src/pages/components/typography.mdx:20]: Is this the right terminology? I've typically heard these referred to simply as `block quote`.

File: src/pages/components/typography.mdx [hunk 2]
```diff
@@ -16,6 +17,17 @@ components:
...
+
+Use Blockquote to represent a body of text that is from another source.
```
Comment (allisonacs) [inline src/pages/components/typography.mdx:22]: Another use case is to bring special emphasis to content found within the body. That's editorial, and we may never see anything like that in product… should it be called out anyways, or would it just cause too much confusion? @steue @vsrixyz

File: src/pages/components/typography.mdx (hunk 1 above)
Comment (jzempel) [reply src/pages/components/typography.mdx:20]: @allisonacs please see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote where we learn this is the "Block Quotation" element. But I'm fine to update, if necessary.

Comment (allisonacs) [reply src/pages/components/typography.mdx:20]: huh, TIL. 

I understand wanting to mirror the standard, but it feels like esoteric language to me. My personal vote is to use a more colloquial term for it, but I get that I am kind of picking/choosing where I want to apply that standard. Up to you 🙂

Comment (jzempel) [reply src/pages/components/typography.mdx]: colloquial is king 👑

**Review by hzhu — APPROVED** [review]

File: src/pages/components/typography.mdx (hunk 2 above)
Comment (jzempel) [reply src/pages/components/typography.mdx:22]: Merging per offline convo with @steue. Happy to adjust copy under a separate PR.

**Pull request merged**